tgui/packages/tgui/styles/themes/paper.scss

```diff
@@ -48,7 +48,10 @@ $font-size: 24px;
   );
 
   // Layouts
-  @include meta.load-css('../layouts/Layout.scss');
+  @include meta.load-css(
+    '~tgui/styles/layouts/Layout.scss',
```

Comment:
Please use a relative path because it's within the same styles folder, no reason not to whatsoever.

Reply:
```suggestion
    '../layouts/Layout.scss',
```